pkg/apis/workspaces/v1alpha1/containerComponent.go

```diff
@@ -8,42 +8,62 @@ type ContainerComponent struct {
 	Endpoints     []Endpoint `json:"endpoints,omitempty"`
 }
 
+// EndpointExposure describes the way an endpoint is exposed on the network.
+// Only one of the following exposures may be specified: public, internal, none.
+// +kubebuilder:validation:Enum=public;internal;none
+type EndpointExposure string
+
+const (
+	// Endpoint will be exposed on the public network, typically through
+	// a K8S ingress or an OpenShift route 
+	PublicEndpointExposure  EndpointExposure = "public"
```

Comment:
It looks like the codebase needs a `go fmt`

```suggestion
	PublicEndpointExposure EndpointExposure = "public"
```

Reply:
Fixed